### PR TITLE
RHDEVDOCS-4687 - edit kubeapidown runbook

### PR DESCRIPTION
**Pull request by bburt-rh**
This PR edits the style and language of this runbook

### CODE REVIEW

File: alerts/cluster-monitoring-operator/KubeAPIDown.md [hunk 1]
```diff
@@ -7,29 +7,32 @@ been reachable by the monitoring system for more than 15 minutes.
...
+responding, and the cluster might be partially or fully non-functional.
 
 ## Diagnosis
 
-Check the status of the API server targets in the Prometheus UI.
+1. Verify the status of the API server targets in Prometheus in the OpenShift
+web console.
 
-Then, confirm whether the API is also unresponsive for you:
+1. Confirm whether the API is also unresponsive:
```
Comment (raptorsun) [inline alerts/cluster-monitoring-operator/KubeAPIDown.md:18]: shall it be `2.` and the line 24 be `3.`? 
or the document generator will give them right numbers?
my markdown render does not count the items correctly.
image

File: alerts/cluster-monitoring-operator/KubeAPIDown.md [hunk 2]
```diff
@@ -7,29 +7,32 @@ been reachable by the monitoring system for more than 15 minutes.
...
 
 ```console
 $ oc cluster-info
 ```
```
Comment (jan--f) [inline alerts/cluster-monitoring-operator/KubeAPIDown.md:23]: Maybe try removing this newline here to fix the item count of the list?

Comment (bburt-rh) [reply alerts/cluster-monitoring-operator/KubeAPIDown.md:23]: Thanks, @jan--f! Had to indent each line of sample code with 4 spaces. That seems to have fixed things. Learn something new every day!

**Review by raptorsun — APPROVED** [review]
/lgtm

**Pull request merged**